Cython/Compiler/Options.py

```diff
@@ -391,7 +391,7 @@ class DEFER_ANALYSIS_OF_ARGUMENTS:
     'fast_gil': ('module',),
     'iterable_coroutine': ('module', 'function'),
     'trashcan' : ('cclass',),
-    'total_ordering': ('cclass', ),
+    'total_ordering': ('cclass', 'class'),
     'dataclasses.dataclass' : ('class', 'cclass',),
```

Comment:
Let's keep the order consistent here, given that both are even right next to each other.
```suggestion
    'total_ordering': ('class', 'cclass'),
    'dataclasses.dataclass' : ('class', 'cclass'),
```